pkg/reconciler/v1alpha1/autoscaling/kpa/kpa_scaler.go

```diff
@@ -114,7 +114,7 @@ func (ks *kpaScaler) Scale(ctx context.Context, pa *pav1alpha1.PodAutoscaler, de
 
 	gv, err := schema.ParseGroupVersion(pa.Spec.ScaleTargetRef.APIVersion)
 	if err != nil {
-		logger.Error("Unable to parse APIVersion.", zap.Error(err))
+		logger.Errorf("Unable to parse APIVersion: %v", zap.Error(err))
```

Comment:
This should be `Errorw` instead I think.

Reply:
Done.